spec/unit/virtus/attribute/collection/value_coerced_predicate_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'spec_helper'
+require 'set'
+
+describe Virtus::Attribute::Collection, '#value_coerced?' do
+  subject { object.value_coerced?(input) }
+
+  let(:object) { described_class.build(Array[Integer]) }
+
+  context 'when input has correctly typed members' do
+    let(:input) { [1,2,3] }
```

Comment:
it's just a style thing but put spaces after comas [source](https://github.com/bbatsov/ruby-style-guide#no-spaces-braces)

`let(:input) { [1, 2, 3] }`